Userland/Libraries/LibJS/Print.cpp

```diff
@@ -384,9 +384,15 @@ ErrorOr<void> print_promise(JS::PrintContext& print_context, JS::Promise const&
 
 ErrorOr<void> print_array_buffer(JS::PrintContext& print_context, JS::ArrayBuffer const& array_buffer, HashTable<JS::Object*>& seen_objects)
 {
+    TRY(print_type(print_context, "ArrayBuffer"sv));
+
+    if (array_buffer.is_detached()) {
```

Comment:
Detaching an array buffer sets its byte length to 0 and accessing it afterwards should not crash - so I'd argue the correct fix is handling detached array buffers in the `byte_length()` getter.

Reply:
(we can still print out the detached state if you want to repurpose this commit)

Reply:
Done, just made a seperate commit to prevent the crash when accessing byte_length from a detached array